### ISSUE TITLE
Share secondary entry point

### ISSUE THREAD

**Issue by mtuzinskiy** [angular-architects/module-federation-plugin#190]
I have a Nx library with secondary entry point.
I'm including it in sharedMappings like this:
```
sharedMappings.register(path.join(__dirname, '../../tsconfig.base.json'), [
  '@xxx/yyy', // my lib
  '@xxx/yyy/zzz' // secondary entry point for my lib
]);
```
 But I'm getting "Can't resolve '@xxx/yyy/zzz'" error. What to do?

**Comment by manfredsteyer** [comment]
Good point. I don't think we cover this case (internal workspace lib with secondary entry point).

**Comment by woppa684** [comment]
We have the same issue (and we spent a week already finding the cause). It didn't even occur to me that MF could be the issue here., We upgraded to NG14 and it just stopped working. Anyway, it would really help if there would be a solution for this!

**Comment by woppa684** [comment]
The Nx module federation seems to have fixed things in this area already by the way https://github.com/nrwl/nx/pull/11286

**Comment by vikashkumar29** [comment]
The same issue also exists with the Angular CLI workspace, with secondary entry points of the workspace library. Also, this happens when we use the path alias defined in tsconfig.json.

Any solution for this will be really great help.

**Comment by rpanadero** [comment]
Is there any update on this? I'm facing the same issue.
